docs/content/docs/faq.md

```diff
@@ -242,3 +242,24 @@ This also applies to Blazor, Xamarin, MAUI, and other mobile frameworks.
 ### What browsers are supported?
 
 {{% include file="_includes/browsers.md" %}}
+
+## Summarization
+
+### Summaries vs Snapshots
```

Comment:
I suggest we remove this header, don't think we need a separate sub-section heading. Instead use the triple # for questions below. Looking over the other questions in the FAQ, the rest of them are also triple #

Reply:
Good call. Updated.